binding/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "pv_rhino"
+version = "0.0.1"
+edition = "2018"
+description = "The Rust bindings for Picovoice's Rhino library"
+license-file = "../../LICENSE"
```

Comment:
does this license get included in deployment? I noticed the pv_porcupine package lists our license as "non-standard". If at all possible ensure that the package either deploys with our license file or a link to Apache-2.0

Reply:
Yes it does. Because we provide the file directly it lists it as "non-standard", otherwise we could just specify `license = "apache-2.0"` if you would prefer that to show up.

Reply:
I would put `license = "apache-2.0"` so that it shows up on crates.io. Non-standard might scare some folks off.